esp/src/eclwatch/nls/es/hpcc.js

```diff
@@ -247,6 +261,7 @@ define(
     FindNext: "Encontrar próximo",
     FindPrevious: "Encontrar previo",
     Finished: "Terminado",
+    FirstN: "PrimeroN",
```

Comment:
I'm not sure about the context, but it seems like this should be PrimerosN
```suggestion
    FirstN: "PrimerosN",
```